README.adoc

```diff
@@ -47,7 +47,7 @@ using, and developing Minishift:
 
 - https://docs.openshift.org/latest/minishift/getting-started/index.html[Getting started with Minishift]
 - https://docs.openshift.org/latest/minishift/using/index.html[Using Minishift]
-- https://docs.openshift.org/latest/minishift/developing/index.html[Developing Minishift]
+- https://docs.openshift.org/latest/minishift/contributing/developing.html[Developing Minishift]
```

Comment:
It should be linked to https://docs.openshift.org/latest/minishift/contributing/index.html and s/Developing Minishift/Contributing to Minishift/. 
https://docs.openshift.org/latest/minishift/contributing/index.html[Contributing to Minishift] 
CC @hferentschik as he did changes around this recently